assets/src/scss/base/_mixins.scss

```diff
@@ -12,6 +12,13 @@
   }
 }
 
+// Mobile to Large (From 576px to 992px)
+@mixin small-to-large {
+  @media (min-width: #{$small-width}) and (max-width: #{$large-width - 1}) {
+    @content;
+  }
+}
+
```

Comment:
The production released behavior is not taking account the behavior for smaller screens because it has set `400px` of width, and if any user navigate with an `iPhone 5`, which is the initially reported device, some parts of the navbar will be hidden (included the X button), since the width of the navbar (`400px`) exceeds the width of the viewport (`320px`).

But *Why I've added this new @media?* 
We need to specify behaviors to this kind of implementations. _It's a very particular issue_.
For one hand, there is a rule `@media medium-and-less` that set the `navbar` to `400px`, on the other hand there is another rule `@media small-and-less`, which it was included on this PR, that set to `100vw` the `navbar` width for smaller screens. In consequence, the `width` attribute is set in two different `@media queries` through the same viewport width and causes this [unexpected behavior](https://github.com/greenpeace/planet4-master-theme/pull/1593/files#r792628708) reported by Flo.

Finally, I assume that when Safari will have support to all these unsupported features in `< OS 13`, we could remove all these unloved [workarounds](https://github.com/greenpeace/planet4-master-theme/pull/1593/files#diff-91009267fa7864d1b148d657a615793b252f3e148c4001b639eab5a4c51324e4R128-R159).